packages/contracts/src/errors.ts

```diff
@@ -51,6 +67,12 @@ export const getFullError = (error: string): string => {
   return fullError;
 };
 
+/**
+ * Gets the contract error from the human readable string
+ *
+ * @param error - Contract error
+ * @returns Human readable translation of the error
+ */
```

Comment:
We need to be consistent - are we doing comments using `/* */` format or `///` ?